### DISCUSSION TITLE
[Request] Swing Music LXC

### DISCUSSION THREAD

**Discussion by x3kim** [tteck/Proxmox#3835] [Ideas]
Hello. :)

First, THANK YOU so much for all of the already pre-built scripts we can use already. I love it!
I'm almost finished with my setup, but im lacking a music service for myself and found this one:

https://github.com/swingmx/swingmusic

Is there any chance this could be added soon or should i learn how to do this by myself since there is not enough interest in it? If so: could someone maybe help to figure out how to install it by myself in an lxc?
Thanks in advance for any answer on this.

**Comment by tteck** [comment]
Have you looked at Navidrome?

**Reply by x3kim** [reply]
Don't know how i missed that / didn't saw that. I am so sorry! This seems like a good alternative for SwingMusic. Thank you tteck!
I'll give it a try. 🧡

**Reply by x3kim** [reply]
Sadly its just a mess with Navidrome. It puts most of my albums in [UNKNOWN] and i don't like the GUI that much. Well.. i guess i have to learn how to write bash scripts by myself then (hope it doesn't take ages). Again thanks for your recommendation and all the scripts. :)

**Reply by Mellowlynx** [reply]
If you use Plex, consider if for your music too.
Together with the Plexamp app on a phone, it works great.
Plex might also be better at indexing the songs and albums and getting the right information about them.

**Comment by x3kim** [comment]
So. I tried it, edited your examples of adguard + adguard-install.sh - but (sry for the dumb question maybe) how the hell do i execute them now? Do i need to upload them to your git or something or do i need to send them to you?

**Reply by tteck** [reply]
Just create a Debian 12 LXC, install FFmpeg, download the binary and execute. Or, run it in Docker.
The program looks lame to me.

**Comment by oOStroudyOo** [comment]
If you have Home Assistant, I'd also consider Music Assistant, found in the Home Assistant add-on store.
It can combine multiple streaming services into 1 including your local files.

https://music-assistant.io/
